test/e2e/cluster_upgrade.go

```diff
@@ -50,18 +51,27 @@ var _ = framework.KubeDescribe("Upgrade [Feature:Upgrade]", func() {
 
 	framework.KubeDescribe("master upgrade", func() {
 		It("should maintain a functioning cluster [Feature:MasterUpgrade]", func() {
+			target, err := framework.RealVersion(framework.TestContext.UpgradeTarget)
```

Comment:
Maybe. I would rather change some of the upgrade functions to take a `version.Version` type instead of a string first. Because right now we need 2 representations of each version.